src/Xrefcheck/Config.hs

```diff
@@ -7,18 +7,18 @@
 
 module Xrefcheck.Config where
 
-import Data.Aeson.Options (defaultOptions)
 import Data.Aeson.TH (deriveFromJSON)
 import Data.Yaml (FromJSON (..), decodeEither', prettyPrintParseException, withText)
 import Instances.TH.Lift ()
 
 -- FIXME: Use </> from System.FilePath
 -- </> from Posix is used only because we cross-compile to Windows and \ doesn't work on Linux
-import System.FilePath.Posix ((</>))
 import Data.FileEmbed (embedFile)
+import System.FilePath.Posix ((</>))
```

Comment:
The comment above seems misplaced now

Reply:
Oh, good catch. A bit sad that such things occur automatically.